src/main/java/ui/listpanel/ListPanelCell.java

```diff
@@ -39,7 +39,12 @@ public void updateItem(TurboIssue issue, boolean empty) {
             return;
         }
         if (!issue.isOpen()){
-            getStyleClass().add("issue-cell-closed");
+            if (!getStyleClass().contains("issue-cell-closed")) {
+                getStyleClass().add("issue-cell-closed");
+            }
+        } else {
+        // prevents the closed issue style class from being applied for open issues when scrolling through list view
+            getStyleClass().remove("issue-cell-closed");
```

Comment:
What does this statement do? Don't explain it in terms of a bug. The bug shouldn't have been there in the first place so it should not be used to explain code.